Content.Server/_CP14/EntityEffects/Effects/AdjustAllSolutionThermalEnergy.cs

```diff
@@ -0,0 +1,55 @@
+using Content.Shared.Chemistry.EntitySystems;
+using Content.Shared.EntityEffects;
+using JetBrains.Annotations;
+using Robust.Shared.Prototypes;
+
+namespace Content.Server.EntityEffects.Effects;
+
+/// <summary>
+///     Adjusts the thermal energy of all the solutions inside the container.
+/// </summary>
+[UsedImplicitly]
+[DataDefinition]
+public sealed partial class AdjustAllSolutionThermalEnergy : EntityEffect
+{
+    /// <summary>
+    ///     The change in energy.
+    /// </summary>
+    [DataField("delta", required: true)] private float _delta;
```

Comment:
```suggestion
    [DataField(required: true)] public float Delta;
```

what's the purpose of making them private? All modern created EntityEffect usually work with public fields